README.md

```diff
@@ -1,15 +1,46 @@
-# 🚀 Welcome to your new awesome project!
+# Leaderboard with webpack and gitflow 
 
-This project has been created using **webpack-cli**, you can now run
+> **This is Leaderboard that you can add and store your score  created using html,css,JavaScript and webpack configuration files and also linters files.In this project I used gitflow to create this project**
 
-```
-npm run build
-```
 
-or
+## Built With 
+- Webpack
+- gitFlow
+- HTML 
+- JavaScript
+- Css
 
-```
-yarn build
-```
 
-to bundle your application
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
```

Comment:
- [ ] Kindly give instructions to the user on how to use the application, like if the user should install the node dependencies with `npm i` and use the `npm run build` command to build the application, and after that he/she can open the live server. This section is missing the steps you mention in it.